docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,150 @@
+# Version 2.15.0 (March 2024)
+
+Welcome to the Zowe Version 2.15.0 release!
+
+See [New features and enhancements](#new-features-and-enhancements) for a full list of changes to the functionality. See [Bug fixes](#bug-fixes) for a list of issues addressed in this release.
+
+**Download v2.15.0 build**: Want to try new features as soon as possible? You can download the v2.15.0 build from [Zowe.org](https://www.zowe.org/download.html).
+
+## New features and enhancements
+
+Zowe Version 2.15.0 contains the enhancements that are described in the following topics.
+
+:::info find out more
+To watch a demo of new enhancements and updated features included in a Zowe minor release, look for the release demo recording in the [Zowe V2 System Demo playlist](https://www.youtube.com/playlist?list=PL8REpLGaY9QGjSTAqZaWxLG_g-jW1qGmo) on YouTube.
+
+System demos are typically held the week after a minor release becomes available. Check the [Open Mainframe Project Calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar) for the latest schedule.
+:::
+
+### Zowe installation and packaging
+
+### Zowe Application Framework
+
+#### Zlux App Server
+
+
+### Zowe API Mediation Layer
+
+- Timeout configuration values of the websocket webclient are now supported in the API Gateway. ([#3315](https://github.com/zowe/api-layer/issues/3315))
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Added `--binary` and `--encoding` options to the `zos-files edit` command in Zowe CLI V2.
+
+- Hid the progress bar if the `CI` environment variable is set, or if the `FORCE_COLOR` environment variable is set to `0`. ([#1845](https://github.com/zowe/zowe-cli/issues/1845))
+
+#### Zowe CLI Imperative Framework
+
+- Added multiple APIs to the `ProfileInfo` class to help manage schemas between client applications. ([#2012](https://github.com/zowe/zowe-cli/issues/2012))
+
+- Enhancement: Hid the progress bar if `CI` environment variable is set, or if `FORCE_COLOR` environment variable is set to `0`. ([#1845](https://github.com/zowe/zowe-cli/issues/1845))
+
+#### CICS Plug-in for Zowe CLI
+
+#### DB2 Plug-in for Zowe CLI
+
+#### MQ Plug-in for Zowe CLI
+
+#### IMS Plug-in for Zowe CLI
+
+- Added notices announcing the deprecation of the IMS Plug-in for Zowe CLI. ([#66](https://github.com/zowe/zowe-cli-ims-plugin/pull/66))
+
+#### z/OS FTP Plug-in for Zowe CLI
+
+- Simplified the preparation for JCL system tests.
```

Comment:
@adam-wolfe - would like to add an issue/pr link to this. would also like to explain the benefit of this change.

Reply:
https://github.com/zowe/zowe-cli-ftp-plugin/pull/151. It looks like the PR should reduce the amount of preparation needed to run system tests, but @zFernand0 might be able to give a good summary.

Reply:
This is something that can be omitted from the Zowe Docs site.
While it does help us deliver higher quality features faster by making the codebase more maintainable, this is not something that the user will be able to perceive (like a new command) with this new version of the plug-in.